src/goto-programs/goto_program.h

```diff
@@ -332,6 +332,13 @@ class goto_programt
       code.make_nil();
     }
 
+    /// Transforms an existing instruction into a skip,
+    /// retaining the source_location
+    void turn_into_skip()
+    {
+      clear(SKIP);
+    }
```

Comment:
`make_skip` exists which already does the same thing

Reply:
Indeed; however, the prefix `make_` is now commonly used for factories, and most programmers will be surprised to see these have an effect on `this`. My hope is that `turn_into_` makes this much clearer.

Reply:
In that case, `make_skip` should be removed (or at least deprecated), and the other `make_` functions should be renamed as well.

Reply:
Yes, coming!